fermipy/sensitivity.py

```diff
@@ -129,6 +129,7 @@ def compute_counts(self, skydir, fn, ebins=None):
         if self._gdiff_fit is not None:
             bkg_fit = []
 
+
         for psf, exp in zip(self._psf, self._exp):
```

Comment:
It's not a big deal but if possible you should try to avoid adding extraneous whitespace as it makes it harder to to review diffs.